### DISCUSSION TITLE
. net core only uses skywalking

### DISCUSSION THREAD

**Discussion by huangyedie** [apache/skywalking#10932] [Q&A]
. net core only uses skywalking. Generally, link information is automatically pushed to ES through middleware. I want to push link information to ES through my own method. What is the structure of skywalking pushing to ES? Can the skywalking link information pushed to ES be displayed through the skywalking UI?

Or there could be a better way to solve this problem because my project engineering is distributed and cannot be managed through service registration and discovery. Therefore, I want to assemble my own skywalking structure to push to ES

**Comment by wu-sheng** [comment]
I can't understand what do you mean by link.

**Reply by huangyedie** [reply]
我用中文来表达吧。
.net使用skywalking 。一般情况下，链路信息是通过中间件自动推送到ES的。我想通过我自己的方法将链路信息推送到ES。skywalking 推向ES的结构是什么？推送给ES的skywalking链接信息是否可以通过skywalking UI显示？

或者有更好的方法来解决这个问题，因为我的项目工程是分布式的，不能通过服务注册和发现来管理，也不是什么微服务，因此，我想组装我自己的skywalking结构，推送到ES，就是链路信息我自己组装推送到es，skywalking  UI 拉取我推送的链路信息展示、

**Comment by huangyedie** [comment]
是不是可以使用OpenTelemetry 这个来自定义链路信息呢？然后把自定义的链路信息推送到es里面。使用skywalking  来展示呢

**Comment by wu-sheng** [comment]
First of all, the discussion channel is only in English. Then, .net agent is not part of the Apache Software Foundation. We can't apply from here.
